frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputForm.tsx

```diff
@@ -127,7 +108,8 @@ function ActionParametersInputForm({
   return (
     <ActionForm
       action={action}
-      initialValues={initialValues}
+      initialValues={values}
+      prefetchesInitialValues={Boolean(fetchInitialValues)}
```

Comment:
nit: `!!` is shorter

Reply:
Shorter does not mean better or worse - it depends.

FYI I stopped using this notation few years ago for these reasons:
1. readability - if a line has more operators than a single boolean cast (`!!`), then it gets more difficult to read
2. verbosity - I've seen `!!` that were actual typos
3. consistency - cast types only in one way throughout the codebase
    - with all boolean casts (you're probably gonna want to `array.filter(Boolean)`, not `array.filter(item => !!item)`
    - with other types casts (I think only casting to `Number` has a similar "trick", e.g. `+"7"`)
4. when working in a team (or on an open source project), people not familiar JS/TS might come and read the code - `!!` is definitely more cryptic than `Boolean()`

I'm fine with this notation (`!!`) being used as long as the code is still readable (see point 1), but unless it's explicitly forbidden I will continue to cast types using type constructors as functions.

Reply:
as for filter, I'd use Boolean for visibility, as for this specific case, I'd use `!!` - just personal preference. But it's `nit`